kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/AdminResource.scala

```diff
@@ -270,6 +275,18 @@ private[v1] class AdminResource extends ApiRequestContext with Logging {
               s"${e.getMessage}")
         }
       }
+
+      if (Option(refId).isDefined) {
```

Comment:
Can we add a new parameter to decide whether to kill forcefully or gracefully？

Reply:
Of course.